### ISSUE TITLE
Why do I call Dynamoid::TransactionWrite  and get an error name uninitialized constant?

### ISSUE THREAD

**Issue by namdv-1375** [Dynamoid/dynamoid#763]
```ruby
[1] pry(main)> Dynamoid::TransactionWrite 
NameError: uninitialized constant Dynamoid::TransactionWrite
from (pry):1:in `__pry__'
```
Why do I call Dynamoid::TransactionWrite  and get an error name uninitialized constant?
Please help me, thankssss <3

**Comment by ckhsponge** [comment]
I thought I remembered the first version of TransactionWrite being released in 3.10 but I don't see it there. It's in main so you can put this in your gemfile:
`gem 'dynamoid', git: 'https://github.com/Dynamoid/dynamoid.git'`

This branch has some more documentation about some hidden features but won't be live until the non-transaction actions catch up:
`gem 'dynamoid', git: 'https://github.com/ckhsponge/dynamoid.git', branch: 'transact-update'`

@andrykonchin Want to release transactions some time?

**Comment by namdv-1375** [comment]
`gem 'dynamoid', git: 'https://github.com/Dynamoid/dynamoid.git'` -> works for me
Thanksssss <3

**Comment by namdv-1375** [comment]
@ckhsponge Let me ask, if I want to `delete_all` with `Dynamoid::TransactionWrite`, is there any way?

**Comment by ckhsponge** [comment]
No. I believe in DynamoDB deletions can only occur individually, in a batch or in a transaction and always require primary keys. Dynamoid's delete_all probably gets keys and deletes in batches and this wouldn't be possible in a transaction.

**Comment by markprovan** [comment]
@andrykonchin Is there any plan for when the next release will take shape?

Happy Friday!

**Comment by andrykonchin** [comment]
This is released in v3.11.0
